CHANGES.md

```diff
@@ -1,4 +1,189 @@
+## stable-2.3.0
+
+### Announcing Linkerd 2.3 🎈
+
+This stable release introduces a new TLS Identity system into the default
+Linkerd installation, replacing `tls=optional` and the `linkerd-ca` controller.
+Now, proxies generate ephemeral private keys into a tmpfs directory and
+dynamically refresh certificates, authenticated by Kubernetes ServiceAccount
+tokens, via the newly-introduced Identity controller.
+
+Now, all meshed HTTP communication is private and authenticated by default.
+
+In the CLI, this release includes 2 new commands:
+* `linkerd metrics` fetches metrics directly from a Linkerd proxy
+* `linkerd upgrade` upgrades your existing Linkerd installation
+
+Among the many improvements to the Web UI, we've added a Community page to
+surface news and updates from linkerd.io.
+
+For more details, see the announcement blog post:
+https://blog.linkerd.io/2019/04/16/announcing-linkerd-2-3/
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Upgrade notes**: The `linkerd-ca` controller is deprecated in favor of the
+`linkerd-identity` controller. If you had previously installed Linkerd with
+`tls=optional`, you will need to manually delete `linkerd-ca` after upgrading.
+Also, `single-namespace` mode is no longer supported. For full details on
+upgrading to this release, please see the
+[upgrade instructions](https://linkerd.io/2/upgrade/#upgrade-notice-stable-2-3-0).
+
+**Special thanks to**: @codeman9, @harsh-98, @huynq0911, @KatherineMelnyk,
+@liquidslr, @paranoidaditya, @Pothulapati, @TwinProduction, and @yb172!
+
+**Full release notes**:
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd control
+    plane to be reinstalled or reconfigured; it is particularly useful for
+    automatically reusing flags set in the previous `install` or `upgrade`
+  * Introduced the `linkerd metrics` command for fetching proxy metrics
+  * **Breaking Change:** The `--linkerd-cni-enabled` flag has been removed from
+    the `inject` command; CNI is configured at the cluster level with the
+    `install` command and no longer applies to the `inject` command
+  * **Breaking Change** Removed the `--disable-external-profiles` flag from the
+    `install` command; external profiles are now disabled by default and can be
+    enabled with the new `--enable-external-profiles` flag
+  * **Breaking change** Removed the `--api-port` flag from the `inject` and
+    `install` commands, since there's no benefit to running the control plane's
+    destination API on a non-default port (thanks, @paranoidaditya)
+  * Removed the `--tls=optional` flag from the `linkerd install` command, since
```

Comment:
I'd also prefix this bullet point with `**Breaking change**`, for consistency with the three bullet points that precede it, which also describe flags being removed.